lib/main_common.pm

```diff
@@ -1105,7 +1105,7 @@ sub load_x11tests {
         # TODO test on openSUSE https://progress.opensuse.org/issues/31972
         if (is_sle && (!is_server || we_is_applicable)) {
             loadtest "x11/eog";
-            loadtest "x11/rhythmbox";
+            loadtest(is_sle('<=12-sp3') ? "x11/rhythmbox" : "x11/gnome_music");
```

Comment:
what's about sle12 sp4? I wonder if we should use condition as >15..

Reply:
I am not sure, if sp4 is going to keep rhythmbox or it will change to gnome-music. Anyway we should not cause any harm using <15. WDYT ?

Reply:
We don't know what will happen, so I would keep in mind current behavior. I personally would go go for '15+' as it looks better, but '<15' will do the trick as well, so it's up to you.

Reply:
For anyone who reads the thread, I've checked and it's rhythmbox in SLE 12 SP4: https://openqa.suse.de/tests/1536211#step/rhythmbox/3

Reply:
Yeah, unless we are talking about the "legacy" module you can expect packages to stay for the lifetime of a code stream, the beauty of enterprise Linux :)